packages/botonic-react/src/components/carousel.jsx

```diff
@@ -1,22 +1,44 @@
-import React, { useContext } from 'react'
+import React, { useContext, useState, useEffect, useRef } from 'react'
 import { Message } from './message'
 import { WebchatContext } from '../contexts'
 import { StyledScrollbar } from '../webchat/components/styled-scrollbar'
 import { isBrowser, INPUT } from '@botonic/core'
 import styled from 'styled-components'
-import { COLORS } from '../constants'
+import { COLORS, WEBCHAT } from '../constants'
+import LeftArrow from '../assets/leftArrow.svg'
+import RightArrow from '../assets/rightArrow.svg'
+import { resolveImage } from '../utils'
 
 const StyledCarousel = styled.div`
-  padding-top: 10px;
+  padding: 10px 0px;
   display: flex;
   flex-direction: row;
   max-width: 100%;
+  overflow-x: auto;
 `
 
 const StyledItems = styled.div`
   display: flex;
 `
 
+const StyledArrowContainer = styled.div`
+  position: absolute;
+  box-sizing: border-box;
+  top: 10px;
+  height: calc(100% - 20px);
+  width: 25px;
+  background: ${COLORS.SOLID_WHITE};
```

Comment:
you can do it transparent by just removing this attribute, it would be cleaner 😜.